data/serializer/src/main/java/tech/pegasys/teku/api/response/v1/teku/DepositTreeSnapshotSchema.java

```diff
@@ -48,13 +56,26 @@ public class DepositTreeSnapshotSchema {
           "Hash of the execution block containing the highest index deposit stored in the snapshot")
   public Bytes32 executionBlockHash;
 
+  @JsonProperty("execution_block_height")
+  @Schema(
+      type = "string",
+      format = "uint64",
+      example = "1",
+      description =
+          "Height of the execution block in canonical chain containing the highest index deposit stored in the snapshot")
+  public UInt64 executionBlockHeight;
+
   @JsonCreator
   public DepositTreeSnapshotSchema(
       @JsonProperty("finalized") final List<Bytes32> finalized,
-      @JsonProperty("deposits") final UInt64 deposits,
-      @JsonProperty("execution_block_hash") final Bytes32 executionBlockHash) {
+      @JsonProperty("deposit_root") final Bytes32 depositRoot,
+      @JsonProperty("deposit_count") final UInt64 depositCount,
+      @JsonProperty("execution_block_hash") final Bytes32 executionBlockHash,
+      @JsonProperty("execution_block_height") final UInt64 executionBlockHeight) {
```

Comment:
I didn't notice this earlier but I think this class should just be called `DepositTreeSnapshot`.  The schema is an SSZ concept and usually our API definition classes as just named after the object they provide in the API without a Schema prefix.